R/q.fnames.R

```diff
@@ -1,12 +1,22 @@
 q.fnames <- function(results, fnames) {
-  # Error checks 
-  if (class(results) != "QmethodRes") stop("The object provided is not of class 'QmethodRes'")
+  # Error checks
+  if (class(results) != "QmethodRes") stop("The object provided is not of class 'QmethodRes'.")
   comb <- array(sapply(fnames, function(x) substring(x,1,1)))
   nos <- 0:9
+  if (!is.vector(fnames)) {  # correct type?
```

Comment:
please replace with the short code version (as above)
'if (!is.vector(fnames)) stop("The factor names specified are not a vector." )'